pkg/devspace/deploy/deployer/kubectl/kubectl.go

```diff
@@ -338,3 +345,60 @@ func (d *DeployConfig) isKustomizeInstalled(ctx context.Context, dir, path strin
 	err := command.Command(ctx, dir, expand.ListEnviron(os.Environ()...), nil, nil, nil, path, "version")
 	return err == nil
 }
+
+func (d *DeployConfig) applyDeployPatches(ctx devspacecontext.Context, resource *unstructured.Unstructured) (*unstructured.Unstructured, error) {
+	out, err := yaml.Marshal(resource)
+	if err != nil {
+		return resource, err
+	}
+
+	patches := patch.Patch{}
+	for idx, kubepatch := range d.DeploymentConfig.Kubectl.Patches {
+		newPatch := patch.Operation{
+			Op:   patch.Op(kubepatch.Operation),
+			Path: patch.OpPath(kubepatch.Path),
```

Comment:
Would we we want to support the older style of paths by adding the transform step that profiles take? It could be confusing for users to have the format accepted in one area, but not the other.

Similar to this [line](https://github.com/devspace-sh/devspace/blob/main/pkg/devspace/config/loader/profile.go#L246).